File: tests/cli/etc/tokentube/conf.d/lua/storage-save.lua
if( type == "user" ) then
	local file = io.open( "boot/tokentube/user/" .. identifier, "w" )
	file:write( data );
	file:close()
end
if( type == "otp" ) then
	local file = io.open( "boot/tokentube/otp/" .. identifier, "w" )
	file:write( data );
	file:close()
end
if( type == "key" ) then
	local file = io.open( "etc/tokentube/keys/" .. identifier, "w" )
	file:write( data );
	file:close()
end
if( type == "helpdesk" ) then
	local file = io.open( "var/spool/tokentube/helpdesk/" .. identifier, "w" )
	file:write( data );
	file:close()
end

File: tests/cli/etc/tokentube/conf.d/python/storage-save.py
#!/usr/bin/python3

import os, sys


mapping = {
	"USER": "./boot/tokentube/user",
	"OTP":  "./boot/tokentube/otp",
	"KEY":  "./etc/tokentube/keys"
}


path = mapping.get( sys.argv[1] );
if( path is not None ):
	f = open( path + "/" + sys.argv[2] + ".dat", "wb+" );
	f.write( data )
	f.close()

